app/services/search/classified_listing.rb

```diff
@@ -3,7 +3,7 @@ class ClassifiedListing < Base
     INDEX_NAME = "classified_listings_#{Rails.env}".freeze
     INDEX_ALIAS = "classified_listings_#{Rails.env}_alias".freeze
     MAPPINGS = JSON.parse(File.read("config/elasticsearch/mappings/classified_listings.json"), symbolize_names: true).freeze
-    DEFAULT_PAGE = 0
+    DEFAULT_PAGE = 1
```

Comment:
If we change these defaults we then have to change what the frontend is passing to the controllers bc they are all passing 0 right now.